assayist/processor/post_analyzer.py

```diff
@@ -12,7 +12,7 @@
 # Do not include the leading root dir because the analyzer works with paths relative to the
 # unpacked content.
 IGNORED_DIRS = (
-    'var/lib/yum/',  # Yum ghost files
+    'var/',  # Any variable files such as logs, caches, or Yum database files
```

Comment:
In theory, we could whitelist more directories here. Not sure whether excluding whole var is a good idea.

DB was dropped but I can see something suspicious even in running builds for few hours
```
neo4j> MATCH (n:UnknownFile) WHERE n.path STARTS WITH "/var/" RETURN distinct(n.path);
+-------------------------------------------------------------------------------------+
| (n.path)                                                                            |
+-------------------------------------------------------------------------------------+
| "/var/lib/rhsm"                                                                     |
| "/var/log/rhsm"                                                                     |
| "/var/log"                                                                          |
| "/var/lib/alternatives"                                                             |
| "/var/spool/mail"                                                                   |
| "/var/lib/rhsm/repo_server_val"                                                     |
| "/var/www/cgi-bin/keystone"                                                         |
| "/var/www/cgi-bin/aodh"                                                             |
| "/var/lib/kolla/venv/bin"                                                           |
| "/var/cache/yum/abcd-12.0-ceph-2-mon"                                            |
| "/var/cache/yum/abcd-rhel-7.6-ha"                                                |
| "/var/cache/yum/abcd-rhel-7.6-server"                                            |
| "/var/cache/yum"                                                                    |
| "/var/cache/yum/abcd-12.0-geph-2-tools"                                          |
| "/var/cache/yum/abcd-rhel-7.6-extras"                                            |
| "/var/cache/yum/ppc64le/7Server/rhel-7-for-power-le-rpms"                           |
| "/var/cache/yum/ppc64le/7Server"                                                    |
| "/var/cache/yum/x86_64/7Server/rhel-7-server-rpms"                                  |
| "/var/cache/yum/x86_64/7Server"                                                     |
| "/var/www/cgi-bin/cinder"                                                           |
| "/var/cache/yum/abcd-12.0-optools-puddle"                                        |
| "/var/cache/yum/abcd-rhel-7.6-extras/gen"                                        |
| "/var/cache/yum/abcd-rhel-7.6-ha/gen"                                            |
| "/var/cache/yum/abcd-rhel-7.6-image-build-override"                              |
| "/var/cache/yum/abcd-rhel-7.6-image-build-override/gen"                          |
| "/var/cache/yum/abcd-12.0-optools-puddle/gen"                                    |
| "/var/cache/yum/abcd-rhel-7.6-server/gen"                                        |
| "/var/cache/yum/abcd-12.0"                                                          |
| "/var/cache/yum/abcd-12.0/gen"                                                      |
| "/var/cache/yum/abcd-12.0-override/gen"                                             |
| "/var/cache/yum/abcd-12.0-override"                                                 |
| "/var/cache/yum/abcd-12.0-geph-2-mon/gen"                                        |
| "/var/cache/yum/abcd-12.0-geph-2-tools/gen"                                      |
| "/var/lib/texmf"                                                                    |
+-------------------------------------------------------------------------------------+
```

IMHO ignoring `/var/www/cgi-bin/` might be dangerous.
```
neo4j> MATCH (n:UnknownFile) WHERE n.path STARTS WITH "/var/www/cgi-bin/" RETURN n;
+---------------------------------------------------------------------------------------------------------------------------------------------------------+
| n                                                                                                                                                       |
+---------------------------------------------------------------------------------------------------------------------------------------------------------+
| (:UnknownFile {checksum: "22fc277662e78880218ec05c38c6a2d9b5856fc01d0c5feeabd80441d66fb1c1", path: "/var/www/cgi-bin/keystone", filename: "main"})      |
| (:UnknownFile {checksum: "259625288d8677787c55f5fe83a5839a48a3d27dbe43d0c446654d8411cffc80", path: "/var/www/cgi-bin/keystone", filename: "admin"})     |
| (:UnknownFile {checksum: "506bb4d2842e4d86a8d65200a0f339271c845865d7aca0522a6f507bc9a05152", path: "/var/www/cgi-bin/aodh", filename: "app.wsgi"})      |
| (:UnknownFile {checksum: "40400a354d3297e3c700479276bbcec944e949bd7f3725be73aa5748e56c4466", path: "/var/www/cgi-bin/cinder", filename: "cinder-wsgi"}) |
+---------------------------------------------------------------------------------------------------------------------------------------------------------+
```

Reply:
Ack, updated the list of ignored dirs to add few other ones but not the entire `/var`.